assets/article.styl

```diff
@@ -1,15 +1 @@
-@require '../components/fillwidth_row'
-@require '../components/artwork_item/stylesheets'
-@require '../components/tooltips'
-@require '../components/share'
-@require '../components/merry_go_round'
-@require '../components/article_figure'
-@require '../components/article/stylesheets'
-@require '../components/grid'
-@require '../components/cta_bar'
-@require '../components/gradient_blurb'
-@require '../components/jump'
-@require '../components/email/stylesheets'
-@require '../components/articles_grid/stylesheets'
 @require '../apps/article/stylesheets'
```

Comment:
Once upon a time Stylus didn't have a `@require` feature that would automatically know how to de-duplicate `@include`s. Now that we're using `@require` we can better modularize our stylus files in the same way we expect `require`s to work with Browserify. Hence why I'm moving these requires out of the "entry point" of this asset file and into the components and apps themself.